### DISCUSSION TITLE
Where to report minor typos in online documentation.

### DISCUSSION THREAD

**Discussion by mosqu-ito** [SpenceKonde/megaTinyCore#754] [Q&A] [answered]
Hi, I'm guessing you don't want every little typo to have it's own thread in here, and I suspect most regular Github users already know the answer to this but... I'm not a regular Github user and I can't find the answer easily so I've resorted to asking here.  Thanks.

**Comment by mechatroniks-git** [comment, marked as answer]
If you are not a regular Github user, this is actually a good way to learn, if that's of any interest to you.

The best way is to make the edit yourself. To do this, you need to **fork** the repo (this puts a copy in your GitHub repo), make the spelling fixes on your repo (edit the file), **commit** the file(s) you changed to your repo (saving the changes to your repo), then submit a **pull request** (requesting SpenceKonde to pull in the changes in your repo).  You can even do this all on github.com, if you want.  Other options like Github Desktop is easy to learn, but command line is a bit of a learning curve. 

See some past Pull-Request examples: https://github.com/SpenceKonde/megaTinyCore/pulls?q=is%3Apr+spelling

**Reply by mosqu-ito** [reply]
Yikes, that's terrifying.  What if I screw it all up?  The Great One will kill me in my sleep with just the power of his huge brain!  Okay, I'll carefully study your instructions and see if maybe, just maybe, I can muster the courage.  I can always wrap myself in tinfoil before I go to bed at night.

**Reply by SimonMerrett** [reply]
If you screw it up, Spence will probably let you know in the pull request comments. Hopefully he'll tell you specifically what he would like changed before he's happy to merge your pull request. Nothing to be afraid of and all minor improvements are valuable and make a cumulative difference.

**Reply by mosqu-ito** [reply]
I came running back here eager to make a real contribution...

Logic.cpp

  // Set inputs modes
  //block.LUTCTRLB = (input1 << CCL_INSEL1_gp) | (input0 << CCL_INSEL0_gp);
  block.LUTCTRLB = ((input1&0x0F) << CCL_INSEL1_gp) | ((input0&0x0F) << CCL_INSEL0_gp);


It's already been fixed.   :D
Argh... back to looking for typos I guess.

**Reply by SpenceKonde** [reply]
Yeah - thanks for the nudge on getting that issue out the door, it's a rather nasty bug]

**Comment by mechatroniks-git** [comment]
Not to worry about screwing up the PR, just make sure you don't undo this setting. This allows Spence can go into the PR, make a quick fix if he doesn't like something, then merge.

![image](https://user-images.githubusercontent.com/57600622/182982493-48800dcc-8c86-458b-9994-243853f58357.png)
